src/screencast_keys/preferences.py

```diff
@@ -383,12 +389,14 @@ def draw(self, context):
             if self.show_last_operator:
                 col.prop(self, "last_operator_show_mode")
 
+            layout.separator()
+
+            layout.label(text="UI:")
+            col = layout.column()
+            col.prop(self, "load_on_startup")
```

Comment:
I think this option is not related to UI, so it is appropriate to move to after `layout.separator` located on L356.

Reply:
Changed.